docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -78,7 +79,7 @@ To update the metadata and playback information on the server side, and to retri
         MediaControlServer.Stop();
         ```
 
-- To retrieve the metadata and playback information on the client side, follow these steps:
+- To retrieve the playback information on the client side, follow these steps:
```

Comment:
In **line no: 10**, it says: 

"You can [update the metadata and playback information](#updating-and-retrieving-information) on the server side, and then retrieve that information on the client side."

But here in **line no: 82**,  it says "To retrieve the playback information on the client side, follow these steps:"

I can see that "metadata and" is removed. Therefore, should we update the same in **line no:** 10?
Please verify and clarify this doubt for me. Thanks in advance.

Reply:
Done